dev-docs/source/Testing/SANSGUI/ISISSANSGUITests.rst

```diff
@@ -165,10 +165,10 @@ In the ``Beam centre`` tab:
 
 #. Make a note of the four values representing the front/main detector centre positions.
 #. Check that detector is set to ``main-detector`` and click run.
-#. Check the values in the first row (Centre Position - Rear) have changed on completion.
-#. Change the detector to ``Hab`` and re-run ensuring only the values for the front has changed.
-#. For both a plot should appear, as the centre finder is running  with four lines.
-#. The four lines should gradually get closer together.
+#. A plot should appear - make sure to show it if it is behind a another window. It should be updated with four lines, which gradually get closer together. This might take a while to run.
```

Comment:
```suggestion
#. A plot should appear - make sure to show it if it is behind another window. It should be updated with four lines, which gradually get closer together. This might take a while to run.
```